src/test/java/Homework16Test.java

```diff
@@ -0,0 +1,32 @@
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.annotations.Test;
+
+
+
+public class Homework16Test extends BaseTest{
+
+
+    @Test
+    public  static void Navigation () {
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--remote-allow-origins=*");
+        options.addArguments("--disable-notifications");
+
+        WebDriver driver = new ChromeDriver();
```

Comment:
Please finish the homework!You got this problem, please correct it. 

![image](https://github.com/Test-Pro-Bootcamp/A44-QA-Automation-Framework/assets/104761482/f0864ad4-6cb3-45c2-a69c-62ca4850465a)

HINT

`WebDriver driver = new ChromeDriver(options);`